src/subtitle-button/subtitle-button.liquid

```diff
@@ -1,9 +1,14 @@
 {% if hasSubtitles and enableSubtitles %}
   <div class="button-container">
-    <button class="subtitle-button{% if subtitleLocale != empty %} subtitle-button-active{% endif %}" aria-controls="subtitleOptions" aria-label="{{"closed_captioning"|translate}}" label="{{"closed_captioning"|translate}}" tabindex="1050">
-      
+    <button
+      class="subtitle-button{% if subtitleLocale != empty %} subtitle-button-active{% endif %}"
+      aria-controls="subtitleOptions"
+      aria-label="{{initialSubtitleAriaLabel|translate}}"
+      label="{{"initialSubtitleAriaLabel"|translate}}"
+      tabindex="1050">
+
       <span class="hide-visually">
```

Comment:
I would have a question for this. When are we using this hide-visually span?

Reply:
I guess the purpose is to have button text for the browser to index, but not to show -- and guess it's down to labeling stuff pre aria.

Reply:
alright, we need it in this case. thanks for the answer